var/spack/repos/builtin/packages/pnfft/package.py

```diff
@@ -25,46 +25,46 @@ def configure(self, spec, prefix):
 
         configure = Executable('../configure')
 
-        if '+double' in spec['fftw']:
+        if 'double' in spec['fftw'].variants.get('precision'):
             with working_dir('double', create=True):
                 configure(*options)
-        if '+float' in spec['fftw']:
+        if 'float' in spec['fftw'].variants.get('precision'):
             with working_dir('float', create=True):
                 configure('--enable-float', *options)
-        if '+long_double' in spec['fftw']:
+        if 'long_double' in spec['fftw'].variants.get('precision'):
             with working_dir('long-double', create=True):
                 configure('--enable-long-double', *options)
```

Comment:
Same question and suggestion made for the `nfft` package applies here and for the `build`, `check`, and `install` methods below.